app/models/goal.py

```diff
@@ -2,4 +2,12 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(80))
+    tasks = db.relationship("Task", back_populates="goal", lazy=True)
+
+    def to_dict(self):
+        return {
+            "id": self.goal_id,
+            "title": self.title
+        }
```

Comment:
Great helper function! Remember to always put a newline at the end of the file -- this is good style practice in Python and a requirement in some other languages.